core/hydromet_JSON_to_DSS_v2.py

```diff
@@ -0,0 +1,83 @@
+from hydromet import*
```

Comment:
This is a duplicate main function, except for the conditional statements checking whether the BCN contains a "D" or an "L". Delete this function and add an additional argument to the original hydromet_JSON_to_DSS.py to avoid duplicating code.

Reply:
Create one JSON per (hydrology) H01 through H04 and then run the DSS conversion. Then run the DSS conversion at the end. Make sure events are relabeled with the first number being the hydrology number.

Reply:
Resolved, the original function was modified.

Reply:
Ok, the original function was modified.

Reply:
We modified the original function.